mxcubeweb/server.py

```diff
@@ -72,6 +75,8 @@ def init(cmdline_options, cfg):
         )
         Server.flask_socketio.init_app(Server.flask)
 
+        Server.limiter = init_limiter(Server.flask)
```

Comment:
```suggestion
        if app.config.get("RATE_LIMITER_ENABLED", False):
            Server.limiter = init_limiter(Server.flask)
```

Reply:
Perhaps something like that ?

Reply:
It will probably not work if we want to use the decorator, but I thought that we could avoid using the decorator and add something general on `Bluprint` or `Server` level. I think the decorator approach is to fine grained for the moment